src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -276,7 +276,9 @@ private void testAddAction() throws Exception {
 
         feedbackPage.addFeedbackSessionWithTimeZone(
                 newSession.getFeedbackSessionName(), newSession.getCourseId(),
-                null, null, null, null,
+                // explicitly set start time as browser time zone varies;
+                // follow what a browser would pick (next hour from current time)
```

Comment:
How about `next hour based on system time zone`? `current time` may be a little vague.